src/App.jsx

```diff
@@ -2,13 +2,22 @@ import "./App.css";
 import Homepage from "./views/Homepage/Homepage";
 //import ContactPage from "./views/ContactPage/ContactPage";
 import React from "react";
-//import LoginPage from "./views/LoginPage/LoginPage";
+import LoginPage from "./views/LoginPage/LoginPage";
+import { Routes, Route, BrowserRouter } from "react-router-dom";
+import Reservations from "./views/Reservations/Reservations";
 
 function App() {
   return (
-    <div className="App">
-      <Homepage />
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<Homepage />} />
+          <Route path="loginpage" element={<LoginPage />} />
+          {/*}          <Route path="contactpage" element={<ContactPage />} /> */}
+          <Route path="Reservations" element={<Reservations />} />
```

Comment:
change path (only path, component name should stay as it is ofc) from "Reservations" to "reservations"